corehq/apps/registration/views.py

```diff
@@ -170,6 +171,7 @@ class UserRegistrationView(NewUserNumberAbTestMixin, BasePageView):
 
     @use_jquery_ui
     @use_ko_validation
+    @use_new_analytics
```

Comment:
Wasn't there going to be qa of the new analytics code? I understand wanting to use the new code, just curious.

Reply:
yeah, I did some local QA of this and fixed a bug. The registration view is pretty siloed from everything else.